test/unit/org/apache/cassandra/cql3/ViewSchemaTest.java

```diff
@@ -842,6 +842,14 @@ public void testViewMetadataCQLIncludeAllColumn()
                             expectedViewSnapshot);
     }
 
+    @Test
+    public void testAlterViewIfExists() throws Throwable
+    {
+        execute("USE " + keyspace());
+        executeNet("USE " + keyspace());
```

Comment:
Why twice?

Reply:
thanks for taking time to keep a close eye on this. Removed 848L